tools/build/third_party/validate/cc/BUILD.bazel

```diff
@@ -0,0 +1,53 @@
+# Copyright (C) 2019 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("//tools/build:rules.bzl", "android_dynamic_library", "cc_copts")
+
+cc_library(
+    name = "cc",
+    srcs = glob(["*.cpp"]),
+    hdrs = glob(["*.h"]),
+    copts = cc_copts() + select({
+        # Android
+        "//conditions:default": ["-DVK_USE_PLATFORM_ANDROID_KHR"],
+    }) + [
```

Comment:
if the only case in the select is the default case, what's the point of the select?